.github/workflows/backend.yml

```diff
@@ -10,7 +10,7 @@ on:
 # https://docs.github.com/en/actions/using-jobs/using-concurrency#example-using-a-fallback-value
 concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
-  cancel-in-progress: true
+  cancel-in-progress: false
```

Comment:
This is only set for obtaining test results. This will be removed prior to merging.